composables/httpd.ts

```diff
@@ -0,0 +1,22 @@
+const PER_PAGE = 100
+
+function createHttpdBaseURL(domainName = 'seed.radicle.xyz') {
+  return `https://${domainName}/api/v1`
+}
+
+interface FetchOptions {
+  domainName?: string
+}
+
+interface UseFetchIssuesOptions extends FetchOptions {
```

Comment:
Usually cleaner to let those either inferred (if the linter allows) or inline them, especially when it's as simple as in this case. In this case, it's not readily apparent why we separate FetchOptions either. Further than that, there's already a `FetchOptions` defined by `ofetch` which is what powers the `useHttpdFetch` so not a great idea to overload that name.

Reply:
Makes sense. (File deleted)